plugins/module_utils/_api_data.py

```diff
@@ -2390,7 +2390,7 @@ def join_path(path):
     ),
     ('ip', 'traffic-flow', 'target'): APIData(
         unversioned=VersionedAPIData(
-            single_value=True,
+            single_value=False,
```

Comment:
Is this something that was already wrong in the past? If not, you probably also need to make this dependent on the ROS version.

Reply:
This was wrong in the past already. I myself added this error in PR #192.